<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registry_interface</class>
 <widget class="QWidget" name="registry_interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_searched_text">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_search">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_refresh">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>All Items</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTreeWidget" name="treeWidget_registry">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>480</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QTreeWidget {

                selection-background-color: #87CEFF;
                selection-background-color: #0069EB;
                selection-color:white;
                }

                QTreeWidget::item{
                height: 20px; 
                }

                QTreeWidget::item:selected:!active {
                color:white;
                background-color:  #a1c9f7;
                }


                QTreeWidget {
                selection-background-color: #0069D7;
                }


                ScrollBar{
                width:8px;
                }

                QTreeWidget::indicator:checked {background-color:#777777;border: 1px solid #d6d6d6;}
                QTreeWidget::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}

                QTreeWidget::indicator {
                width: 8px;
                height: 8px;
                }</string>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string>Node Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Node Timestamp</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget_registry">
             <property name="styleSheet">
              <string notr="true">
                QTableView::item:selected:!active {
               color:white;
                background-color:  #a1c9f7;
                }

                QTableView::indicator:checked {background-color:black;border: 1px solid #d6d6d6;}
                QTableView::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}

                QTableView::indicator {
                width: 13px;
                height: 13px;}</string>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Node Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Node Timestamp</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Key</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Value</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Hex Value</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
             <column>
              <property name="text">
               <string/>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_searched_items">
        <attribute name="title">
         <string>Searched Items</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tableWidget_searched_items">
           <property name="styleSheet">
            <string notr="true">

                QTableView::item:selected:!active {
               color:white;
                background-color:  #a1c9f7;
                }

                QTableView::indicator:checked {background-color:black;border: 1px solid #d6d6d6;}
                QTableView::indicator:unchecked {background-color:#f1f1f1;border: 1px solid #777777;}

                QTableView::indicator {
                width: 13px;
                height: 13px;}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Node Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Node Timestamp</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Key</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Type</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Hex Value</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>File Path</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Node Path</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
           <column>
            <property name="text">
             <string/>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
